docs/contributing/github-enterprise-auth-from-dev-build.md

```diff
@@ -0,0 +1,14 @@
+# Authenticating GitHub Enterprise from a Desktop-dev Build
+
+So you want to do some testing using GitHub Enterprise in your dev build of GitHub Desktop... here's how!
+
+"GitHub Desktop" is an authorized OAuth app for ghe.io allowing us to login from stable and beta instances of Desktop. However, "GitHub Desktop-dev" is not authorized. So, in order to sign into GitHub Enterprise Server from our development build, we must go through a few extra steps:
+
+1. Find or create an organization on the GHE instance that you are an owner of
+1. [Create a new OAuth app](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/) in the organization. Input `x-github-client://oauth` as the authorization callback URL:
+   <img width="558" alt="New_OAuth_Application" src="https://user-images.githubusercontent.com/7910250/63631725-2ffd8200-c5e0-11e9-92e4-b2e5b61d9c89.png">
+1. Insert the Client ID and Client Secret into the [`app-info.ts` file](https://github.com/desktop/desktop/blob/e3991a8c73ab10ca12fcad23f7e367707051d985/app/app-info.ts#L28-L31)
+1. Build the app
```

Comment:
Since the next step is to be performed within the newly built Desktop-dev, perhaps this step should be worded like “Build and run the Desktop-dev app” to reduce ambiguity around which "app" are we talking about?

Alternatively, this step could say “Build the Desktop-dev app” and the next step could begin with “Within Desktop-dev, go to…”